### PR TITLE
Shcharbunou: lesson01

### PR DESCRIPTION
**Pull request by ddshcharbunou**
close: #37

### CODE REVIEW

File: shcharbunou/README.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+= Shcharbunou
```
Comment (rakovets) [inline shcharbunou/README.adoc:1]: Гэты файл не датычны да гэтага PR

File: shcharbunou/lesson01/build.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+cls
+dir /s /B *.java > sources.txt
+javac -d build_calculator @sources.txt
+echo Main-Class: calculator.application.Runner > manifest.txt
+dir build_calculator /s /B *.class > classes.txt
+set path=C:\Users\danie\.jdks\corretto-11.0.13\bin
+jar cvfm Calculator.jar manifest.txt @classes.txt
+java -jar Calculator.jar
```
Comment (rakovets) [inline shcharbunou/lesson01/build.bat:8]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

File: shcharbunou/lesson01/calculator/application/Runner.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.application;
+
+public class Runner {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/application/Runner.java:4]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/application/CalculatorApplication.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.calculator_service.CalculatorServiceImpl;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.calculator_service.calculator_behavior.CalculatorService;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.client_service.ClientService;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.client_service.client_behavior.Client;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils.Messages;
+
+import java.util.Scanner;
+
+public class CalculatorApplication {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/application/CalculatorApplication.java:13]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/exceptions/DivisionByZeroException.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.exceptions;
+
+public class DivisionByZeroException extends Exception {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/exceptions/DivisionByZeroException.java:4]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/services/calculator_service/CalculatorServiceImpl.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.calculator_service;
+
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.exceptions.DivisionByZeroException;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.calculator_service.calculator_behavior.CalculatorService;
+
+public class CalculatorServiceImpl implements CalculatorService {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/services/calculator_service/CalculatorServiceImpl.java:7]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/services/client_service/ClientService.java [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.client_service.client_behavior.Client;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils.ApplicationReservedWords;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils.Messages;
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils.Operators;
+
+import java.util.Scanner;
+
+public class ClientService implements Client {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/services/client_service/ClientService.java:11]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/services/calculator_service/calculator_behavior/CalculatorService.java [hunk 8]
```diff
@@ -1,6 +1,9 @@
-package by.zhigalko.dev;
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.calculator_service.calculator_behavior;
+
+import by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.exceptions.DivisionByZeroException;
 
 public interface CalculatorService {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/services/calculator_service/calculator_behavior/CalculatorService.java:6]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/services/client_service/client_behavior/Client.java [hunk 9]
```diff
@@ -0,0 +1,11 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.services.client_service.client_behavior;
+
+import java.util.Scanner;
+
+public interface Client {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/services/client_service/client_behavior/Client.java:6]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/services/client_service/client_behavior/Client.java [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+    double inputOperand(Scanner scanner);
+    String inputOperator(Scanner scanner);
+    void printAnswer(double answer);
+    String inputClientAnswer(Scanner scanner);
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/services/client_service/client_behavior/Client.java:10]: "Violation: There isn't separating empty line
Solution: Add blank line"

File: shcharbunou/lesson01/calculator/utils/ApplicationReservedWords.java [hunk 11]
```diff
@@ -0,0 +1,17 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils;
+
+public enum ApplicationReservedWords {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/utils/ApplicationReservedWords.java:4]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/utils/Messages.java [hunk 12]
```diff
@@ -0,0 +1,24 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils;
+
+public enum Messages {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/utils/Messages.java:4]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/utils/Messages.java [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+    INPUT_OPERAND_ERROR_MESSAGE("Error: Value is not double type."),
+    INPUT_OPERATOR_ERROR_MESSAGE("Error: Value is not operator."),
+    INPUT_CLIENT_ANSWER_ERROR_MESSAGE("Error: Unknown command"),
+    UNKNOWN_ERROR_MESSAGE("UNKNOWN ERROR!"),
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/utils/Messages.java:9]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: shcharbunou/lesson01/calculator/utils/Operators.java [hunk 14]
```diff
@@ -0,0 +1,16 @@
+package by.it.shcharbunou.jd1.jd1_1.homework.jd1_1_2.calculator.utils;
+
+public enum Operators {
+
```
Comment (rakovets) [inline shcharbunou/lesson01/calculator/utils/Operators.java:4]: "Violation: Useless blank lines
Solution: Remove blank lines"

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix

File: shcharbunou/lesson01/build.bat (hunk 2 above)
Comment (ddshcharbunou) [reply shcharbunou/lesson01/build.bat]: fixed

**Comment by ddshcharbunou** [comment]
everything fixed

**Pull request closed without merging**